python/tank/platform/engine.py

```diff
@@ -123,7 +122,7 @@ def __init__(self, tk, context, engine_instance_name, env):
         
         # create invoker to allow execution of functions on the
         # main thread:
-        self._invoker = self.__create_main_thread_invoker()
+        self.__create_invokers()
```

Comment:
Sorry to chime in, but as we follow TK coding conventions in the Pipe team, I'd like to second Manne's opinion. Benefits are :
- Clear list of all members
- If the "set initial values" method is moved outside of the __init__ method, members are still always there with a default value
Just my 2 cents